code/obj/ToSplit/barber_shop.dm

```diff
@@ -64,9 +88,16 @@
 
 	New()
 		..()
-		src.setItemSpecial(/datum/item_special/double)
+		src.setItemSpecial(/datum/item_special/double) // should be doable even in barbermode
+		AddComponent(/datum/component/transfer_on_attack)
+		AddComponent(/datum/component/barber/shave)
+		AddComponent(/datum/component/toggle_tool_use)
 		BLOCK_KNIFE
 
+	attack(mob/M as mob, mob/user as mob)
+		if (scalpel_surgery(M, user))
+			return 1
```

Comment:
Is not calling parent intended?

Reply:
Mustve missed that, made razors not work. Now its there and it does!